### DISCUSSION TITLE
Documentação para requisição de nota de empenho

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#149] [Geral]
Olá! Tudo bem?

O TRT3 solicitou uma declaração negativa de trabalho infantil e nepotismo assinada pela pessoa representante da APyB, isto é, nossa excelentíssima presidente @anicelysantos
O modelo que eles enviaram está nesse link:
https://docs.google.com/document/d/1k6I4lap3v9uCURGV_OW0H8rfcgSEu2hM/edit?usp=sharing&ouid=105012949324751328317&rtpof=true&sd=true

Também precisaremos de uma cópia do seu documento de identidade para a validação por parte da instituição. 

Muuuito obrigada <3

**Comment by anadulce** [comment]
@apyb/diretoria poderiam verificar esta solicitação, por favor?

**Comment by anicelysantos** [comment]
@anadulce aqui está: https://drive.google.com/drive/folders/1Zbep4hm191PogX4XXoff-uhqxlB3aPJX?usp=drive_link
Dei acesso a teu email :)
